ci-operator/step-registry/acm/fetch-managed-clusters/acm-fetch-managed-clusters-commands.sh

```diff
@@ -3,6 +3,18 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+
+# The variables defined in this step come from files in the `SHARED_DIR` and credentials from Vault.
+SECRETS_DIR="/tmp/secrets"
+
+# Get the creds from ACMQE CI vault and run the automation on pre-exisiting HUB
+SKIP_OCP_DEPLOY=$(cat $SECRETS_DIR/ci/skip-ocp-deploy)
```

Comment:
Maybe we could find some way to check if its a rehearsal job or not, then when its not a rehearsal job just never run this bit of code. Though I'm not sure if that's possible at the moment.

Reply:
yes, that's a good point Adam, better to not merge it this way.
What about keeping everything as is and only set SKIP_OCP_DEPLOY to false in the code
SKIP_OCP_DEPLOY=false

Reply:
We need to remove the SKIP_OCP_DEPLOY from vault if we do this

Reply:
Do we really need to remove it?
what I will do is the following:

1. Comment out the current Skip ocp deploy
#SKIP_OCP_DEPLOY=$(cat $SECRETS_DIR/ci/skip-ocp-deploy)

2. Add a new line that set its value to false 
SKIP_OCP_DEPLOY=false

Reply:
If we include SKIP_OCP_DEPLOY=false/true in the ref then there will never be a need to use `SKIP_OCP_DEPLOY=$(cat $SECRETS_DIR/ci/skip-ocp-deploy)` which is why I'm thinking we should remove so that we don't confuse anyone, making them think they should use `SKIP_OCP_DEPLOY=$(cat $SECRETS_DIR/ci/skip-ocp-deploy)`

If we want to use an external image from here on we will just change `SKIP_OCP_DEPLOY=false` to `SKIP_OCP_DEPLOY=true` in the ref before running a rehearsal.

Reply:
makes sense.
I'll remove it completely then